gui/src/routes/home/_l.tsx

```diff
@@ -15,7 +15,10 @@ export const Route = createFileRoute("/home/_l")({
 
 function HomePageLayout() {
   useNoticeAlchemyKeyMissing();
-  useNoticeNewVersion();
+  const isDev = import.meta.env.MODE === "development";
+  if (!isDev) {
+    useNoticeNewVersion();
+  }
```

Comment:
just for cleanliness, I'd prefer to have this condition inside the `useNoticeNewVersion` instead of around it